app/src/main/java/com/infomaniak/mail/ui/bottomSheetDialogs/SnoozeBottomSheetDialog.kt

```diff
@@ -55,7 +56,8 @@ class SnoozeBottomSheetDialog @Inject constructor() : SelectScheduleOptionBottom
 
     override fun onCustomScheduleOptionClicked() {
         if (navigationArgs.isCurrentMailboxFree) {
-            openMyKSuiteUpgradeBottomSheet(SCHEDULED_CUSTOM_DATE)
+            trackSnoozeEvent(CUSTOM_SCHEDULE)
+            openMyKSuiteUpgradeBottomSheet(SNOOZE_CUSTOM_DATE)
```

Comment:
Why not `SCHEDULED_CUSTOM_DATE` anymore ?

Reply:
That's what the matomo document envisioned. It should actually never have been SCHEDULED_CUSTOM_DATE inside the snooze bottom sheet to begin with